### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by gienkov**
It's not fully complete yet, and i'm not sure if it requires some additional file edits to accommodate new translation.

### CODE REVIEW

**Comment by gienkov** [comment]
I've decided to not translate materials/data names because FDev does not plan to implement polish translation, and most of polish pilots also prefer english names for these.

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -25,2359 +25,2756 @@
       "Name": "Русский",
       "NotReadyWarning": "Этот перевод является неполным! Если вы выберете его - будет переведена только часть приложения. Недостающий перевод будет отображаться на Английском языке."
     }
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json]: missing a comma after the `}`

**Comment by gienkov** [comment]
Done and done :)

**Comment by msarilar** [comment]
great job, thanks !

**Pull request merged**